openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -70,35 +77,26 @@ def send_grade_to_credentials(
         percent_grade (float): Number representing the learner's grade in this course run
         grade_last_updated (string): String describing the last time this grade was modified in the LMS
     """
-    logger.info(f"Running task send_grade_to_credentials for username {username} and course {course_run_key}")
+    data = {
+        'username': username,
+        'course_run': course_run_key,
+        'letter_grade': letter_grade,
+        'percent_grade': percent_grade,
+        'verified': verified,
+        'lms_last_updated_at': grade_last_updated
+    }
+    logger.info(f"Running task `send_grade_to_credentials` for username {username} with data: {data}")
 
-    countdown = 2 ** self.request.retries
     course_key = CourseKey.from_string(course_run_key)
+    credentials_client = get_credentials_api_client(User.objects.get(username=settings.CREDENTIALS_SERVICE_USERNAME))
+    api_url = urljoin(f"{get_credentials_api_base_url(org=course_key.org)}/", "grades/")
 
-    try:
```

Comment:
wow, this is a really good change, and it wasn't something we identified before. This overbroad try/except is not great structure at all and I'm glad the new decorator makes it easy to fix.